src/utils.js

```diff
@@ -63,9 +67,35 @@ const createFilePayload = (filePath) => {
     file.width = dimensions.width;
     file.height = dimensions.height;
   }
+
+  if (isVideo(filePath)) {
+    const thumbnail = createVideoThumbnail(filePath);
+    file.thumbnail = new URL(`file://${thumbnail}`).toString();
+    console.log(file);
+  }
+
   return file;
 };
 
+const createVideoThumbnail = (file) => {
+  try {
+    const size = "320x180";
+    const folder = path.join(app.getPath("appData"), "JWPlay", "thumbnails");
```

Comment:
Do you want to save the thumbnails in `jw-play` or `JWplay`, or `JW Play`? There's three directories being used.

Reply:
Hum, cannot remember 🤔 

 It was working in dev mode but  but I hit a wall trying to pack ffmpeg and never came back to this PR.

Reply:
In a mac, this is an example of a thumbnail file:

```
file:///tmp/JWPlay/thumbnails/0aec9c3b-2ed8-4155-9168-6fa02e8d0dfa-320x180.png
```

Reply:
I renamed the directory to "JW Play" so it uses the same appData directory that was already created.